dev-haskell/text/text-1.2.4.1_rc1.ebuild

```diff
@@ -22,7 +22,7 @@ SRC_URI="https://github.com/haskell/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.g
 LICENSE="BSD-2"
 SLOT="0/${PV}"
 # keep in sync with ghc-8.10.3
```

Comment:
We need `text-1.2.5.0` brought over from `::haskell` with the correct `CABAL_CORE_LIB_GHC_PV` line.